docs/deploy_logging.md

```diff
@@ -0,0 +1,49 @@
+# Deploy openshift-logging on cluster
+
+### Pre-requisites before starting to deploy openshift-logging
```

Comment:
Now I see that you are also skipping one level from here # directly to ### , so you should move everything from here one level less